WebApiCoreSeed.Data.Tests/EF/DatabaseSeedTest.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Data.EF;
+using Xunit;
+
+namespace WebApiCoreSeed.Data.Tests.EF
+{
+    public class DatabaseSeedTest
+    {
+        [Fact]
+        public async Task Initialize_ShouldAnCreateUser()
```

Comment:
let's work on test naming conventions https://vitalflux.com/7-popular-unit-test-naming-conventions/ I think 5, 6, 7 could be a good fit, currently I can't understand the test objective

Reply:
Discussion in https://github.com/MakingSense/WebApiCore-Seed/pull/27#discussion_r188097957.